wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
```

Comment:
I'm puzzled. Why is it that on my system the folders are not lowercase but they are T2L/Code  and T2L/data? 
how do they look on your system?